mteb/evaluation/evaluators/RerankingEvaluator.py

```diff
@@ -102,21 +104,106 @@ def compute_metrics_batched(self, model):
             all_query_flattened = [
                 q for sample in self.samples for q in sample["query"]
             ]
-            all_query_embs = self._encode_unique_texts(
-                all_query_flattened, encode_corpus_func
-            )
+            if self.evaluator_type == "standard":
+                all_query_embs = self._encode_unique_texts(
```

Comment:
It seems like we want to encode unique texts regardless of whether we use miracl or not